lib/elixir/lib/code/formatter.ex

```diff
@@ -2449,4 +2472,43 @@ defmodule Code.Formatter do
   defp has_double_quote?(chunk) do
     is_binary(chunk) and chunk =~ @double_quote
   end
+
+  # Migration rewrites
+
+  @bool_operators [
+    :>,
+    :>=,
+    :<,
+    :<=,
+    :in
```

Comment:
`and` and `or` could be added here?

Reply:
I wish we could, but unfortunately this wouldn't be safe and might break code that is working today, since `false or 42` is valid (also arguably pretty bad) but `not (false or 42)` would raise.

Reply:
ahhh right, i always think that those are `bool, bool -> bool`, forgot it's `bool, any -> any`
i'll just join you in wishing we could then

Reply:
I guess we could try harder in these cases, checking if the right handside itself is a guard/comparison... but I'm not sure it's worth the extra complexity and it won't capture cases like `and enabled?` anyway.

Reply:
I hesitated between the two approaches, not sure which would be better. Maybe yours is more natural indeed.

Reply:
Actually `not` might be better in this case, if we consider that the user explicitly wrote:

```elixir
unless x > @limit do
```

over 

```elixir
if x <= @limit do
```

which they could have done in the first place, then

```elixir
if not x > @limit do
```

might read better to them for some reason?

It is not critical though, people might see the diff and fix it to their preference eventually.

Reply:
yeah, i think you're right. if not` is a simpler change to reason over than `if $swapped_equality_operator`. thanks for spelling that scenario out.